pyemma/coordinates/data/_base/transformer.py

```diff
@@ -115,7 +115,7 @@ class StreamingTransformer(Transformer, DataSource, NotifyOnChangesMixIn):
         the chunksize used to batch process underlying data.
 
     """
-    def __init__(self, chunksize=1000):
+    def __init__(self, chunksize=None):
```

Comment:
similar to the previous remark: here you default the argument to None (which probably is what is desired)